packages/web/components/pool-detail/concentrated.tsx

```diff
@@ -364,6 +364,12 @@ const ChartHeader: FunctionComponent<{
 const Chart: FunctionComponent<{
   config: ObservableHistoricalAndLiquidityData;
 }> = observer(({ config }) => {
+  if (config?.pool?.concentratedLiquidityPoolInfo) {
+    config.setLastChartData(
+      Number(config.pool.concentratedLiquidityPoolInfo.currentPrice)
+    );
+  }
```

Comment:
We should wrap this in a useEffect to only set it on initial render, and pass in `config?.pool?.concentratedLiquidityPoolInfo` & `config` in the dependency array of useEffect (second param)

Reply:
Uh, idk what useEffect does or means 😅